demonstrations/tutorial_learning_dynamics_incoherently.metadata.json

```
@@ -0,0 +1,66 @@
+{
+    "title": "Learning dynamics incoherently",
+    "authors": [
+        {
+            "id": "diego_guala"
```

Note, will need to update this in metadata as the profile linking PRs have been merged https://github.com/XanaduAI/pennylane.ai-react/pull/1618 and https://github.com/PennyLaneAI/qml/pull/1181
```suggestion
            "username": "Diego"
```